client/mass/charts.js

```diff
@@ -254,6 +255,10 @@ function getChartTypeList(self, state) {
 			}
 		}
 	]
+	if (state.vocab.dslabel.includes('Profile')) {
```

Comment:
this is too arbitrary, shouldn't do it. just add all plot types in `buttons[]` and it's fine

Reply:
Should be solved now, please check in last commit if this is what you meant